time stamp
uint32 current_object
---
time stamp
uint32 	  tray

